subprojects/core-api/src/main/java/org/gradle/api/artifacts/dsl/DependencyHandler.java

```diff
@@ -538,6 +539,30 @@ public interface DependencyHandler extends ExtensionAware {
      */
     <T extends TransformParameters> void registerTransform(Class<? extends TransformAction<T>> actionType, Action<? super TransformSpec<T>> registrationAction);
 
+    /**
+     * Returns the coordinates string of a Gradle plugin, specified by id.
+     *
+     * @param id the id of the plugin
+     * @return the plugin coordinates string
+     *
+     * @since 8.1
+     */
+    @Incubating
+    String plugin(String id);
+
+    /**
+     * Returns the coordinates string of a Gradle plugin,
+     * specified by id and version.
+     *
+     * @param id the id of the plugin
+     * @param version the version of the plugin
+     * @return the plugin coordinates string
+     *
+     * @since 8.1
+     */
+    @Incubating
+    String plugin(String id, String version);
+
```

Comment:
```suggestion
    /**
     * Returns the Maven coordinates string of a Gradle plugin, specified by a plugin identifier.
     *
     * @param id the id of the plugin
     * @return the plugin Maven coordinates string
     *
     * @since 8.1
     */
    @Incubating
    String plugin(String id);

    /**
     * Returns the Maven coordinates string of a Gradle plugin,
     * specified by a plugin identifier and version.
     *
     * @param id the id of the plugin
     * @param version the version of the plugin
     * @return the plugin Maven coordinates string
     *
     * @since 8.1
     */
    @Incubating
    String plugin(String id, String version);

```